addins/Cake.ResxConverter.yml

```diff
@@ -1,8 +1,6 @@
 Name: Cake.ResxConverter
 NuGet: Cake.ResxConverter
 Assemblies:
-- "/**/ResxConverter.Core.dll"
```

Comment:
This is interesting as it could mean types not documented that previously was?

Reply:
The current AddinDiscoverer logic is to only list one assembly in the yml file.